src/config.rs

```diff
@@ -227,6 +227,13 @@ impl Context {
         Ok(self.get_config_int(key).await? != 0)
     }
 
+    pub(crate) async fn get_configured_addr(&self) -> Result<String> {
+        use anyhow::Context;
```

Comment:
is it on purpose to use a `use` statements in function scope? most time i see that atop of files only.

Reply:
I also think, it should be at the top of the file (you can write `use anyhow::Context as _` to not pollute the namespace and get problems with our `Context`)